webapp/src/dogma/features/history/HistoryList.tsx

```diff
@@ -49,21 +43,48 @@ const HistoryList = <Data extends object>({
         cell: (info) => <DateWithTooltip date={info.getValue()} />,
         header: 'Timestamp',
       }),
-      columnHelper.accessor((row: HistoryDto) => row.revision, {
-        cell: (info) => (
-          <NextLink href={`/app/projects/${projectName}/repos/${repoName}/list/${info.row.original.revision}/`}>
-            <Button colorScheme="blue" leftIcon={<FaHistory />} size="sm" onClick={() => handleTabChange(0)}>
-              View Revision {info.row.original.revision}
-            </Button>
-          </NextLink>
-        ),
-        header: 'Actions',
-        enableSorting: false,
-      }),
     ],
     [columnHelper, handleTabChange, projectName, repoName],
   );
-  return <DynamicDataTable columns={columns as ColumnDef<Data>[]} data={data} pagination={true} />;
+  const [{ pageIndex, pageSize }, setPagination] = useState<PaginationState>({
+    pageIndex: 0,
+    pageSize: 10,
+  });
+  const pagination = useMemo(
```

Comment:
Question: Is `useMemo` efficient for this sample case?

Reply:
This is just in case the same (pageIndex, pageSize) gets accessed multiple times.

Reply:
Did you intend to return the same instance for `{ pageIndex, pageSize }`?
`pagination` looks like a simple JSON object.

Reply:
Found the example https://tanstack.com/table/v8/docs/examples/react/pagination-controlled. It looks like `useMemo` is used to stabilise the table between rerendering.

Reply:
Without useMemo
<img width="1175" alt="Screen Shot 2023-01-28 at 00 42 36" src="https://user-images.githubusercontent.com/5079588/215156943-3d0b8f37-d960-4ebf-ad9f-2d97eceed806.png">

With useMemo 
<img width="1181" alt="Screen Shot 2023-01-28 at 00 34 11" src="https://user-images.githubusercontent.com/5079588/215156393-ecb7018b-9af1-4e92-9b36-bfbe337f0f28.png">

Reply:
TIL. 📝